src/applications/vaos/api/index.js

```diff
@@ -169,29 +169,33 @@ export function getSystemDetails(systemIds) {
   }
 
   return promise.then(resp =>
-    resp.data
-      .map(item => ({ ...item.attributes, id: item.id }))
```

Comment:
Another important change, to stop filtering out non-root parent sites from the first question.